test/integration/connect/envoy/case-cfg-router-features/setup.sh

```diff
@@ -59,6 +59,28 @@ routes = [
       prefix_rewrite = "/"
     }
   },
+  {
+    match { http {
+      path_prefix = "/prefix-case-Insensitive/"
+      case_insensitive = true
+      }
+    }
+    destination {
+      service_subset = "v1"
+      prefix_rewrite = "/"
+    }
+  },
+  {
+    match { http {
+      path_prefix = "/prefix-case-INSENSITIVE/"
+      case_insensitive = true
+      }
+    }
+    destination {
+      service_subset = "v1"
+      prefix_rewrite = "/"
+    }
+  },
```

Comment:
Thanks for the tweaks. I think the following config would better exercise the fact that the case insensitivity is working as intended:

```suggestion
  {
    match { http {
      path_prefix = "/prefix-case-insensitive/"
      case_insensitive = true
      }
    }
    destination {
      service_subset = "v1"
      prefix_rewrite = "/"
    }
  },
```